python/rmm/rmm/tests/test_rmm.py

```diff
@@ -1002,3 +1002,10 @@ def test_invalid_logging_level(level):
         rmm.set_flush_level(level)
     with pytest.raises(TypeError):
         rmm.should_log(level)
+
+
+def test_get_free_device_memory():
+    from rmm._lib.memory_resource import get_free_device_memory
+
+    memory = get_free_device_memory()
+    assert memory != 0
```

Comment:
Can we add a nontrivial test, like asserting that free memory decreases after a large allocation using the default CUDA allocator?

Reply:
Done